catalog-rest-service/src/main/java/org/openmetadata/catalog/util/TablesInitializer.java

```diff
@@ -289,16 +292,16 @@ private static void usage() {
 
   private static void printToConsoleInDebug(String message) {
     if (DEBUG_MODE) {
-      System.out.println(message);
```

Comment:
@mohangpatil TablesInitializer is a command line util. We need System.out.println here please revert the changes to the file

Reply:
Sure, thank you for quick reply. I saw it was reported in sonar, so changed it. Reverting it now.